### PR TITLE
Ambience no longer plays in the lobby

### LINKED ISSUE #11198: Space wind plays over lobby music
## Description

I can't properly hear the lobby music because space wind seems to always play. Probably a consequence of being an observer or whatever before the map spawns.

Ideally we should have more ways of controlling ambient music like space wind too so we can have it per tile? Idk if this is possible now but I'll need to make use of it in future. This way you can have maint specific ambient soundtracks, ambient noises for security etc.

### PR DESCRIPTION
**Pull request by DerSheppard**
## Ambience does no longer play in the lobby 

A check prevents ChangeAmbience from calling StartAmbience after the player entity is moved during lobby creation. Also removed unnecessary function call to EndAmbience. 

fixes #11198
doesn't implement the second part of the issue ("Ideally we should have more ways of controlling ambient music like space wind too so we can have it per tile?") though.

**Changelog**

:cl: Sheppard
- fix: Ambience no longer plays in the lobby.

### CODE REVIEW

File: Content.Client/Audio/BackgroundAudioSystem.cs [hunk 1]
```diff
@@ -149,8 +149,8 @@ private void ChangeAmbience(SoundCollectionPrototype newAmbience)
             Timer.Spawn(1500, () =>
             {
                 // If we traverse a few times then don't interrupt an existing song.
-                if (_playingCollection == _currentCollection) return;
-                EndAmbience();
+                // If we are in the Lobby, don't start Ambience because of a player movement
+                if (_playingCollection == _currentCollection || _stateManager.CurrentState is LobbyState) return;
```
Comment (moonheart08) [inline Content.Client/Audio/BackgroundAudioSystem.cs:153]:
```suggestion
                if (_playingCollection == _currentCollection || _stateManager.CurrentState is LobbyState)
                return;
```

**Comment by ShadowCommander** [comment]
You don't need to change anything that you've already done, but just so you know, I have a PR up where I'm working on audio formatting, so try to keep additional changes down so that it's easier to merge. #11238

File: Content.Client/Audio/BackgroundAudioSystem.cs [hunk 2]
```diff
@@ -134,23 +134,26 @@ private void CheckAmbience(TransformComponent xform)
...
             _timerCancelTokenSource.Cancel();
             _currentCollection = newAmbience;
             _timerCancelTokenSource = new();
             Timer.Spawn(1500, () =>
             {
                 // If we traverse a few times then don't interrupt an existing song.
-                if (_playingCollection == _currentCollection) return;
-                EndAmbience();
+                // If we are in the Lobby, don't start Ambience because of a player movement
+                if (_playingCollection == _currentCollection || _stateManager.CurrentState is LobbyState)
```
Comment (metalgearsloth) [inline Content.Client/Audio/BackgroundAudioSystem.cs:155]: Shouldn't this check if it isn't game state?

Comment (DerSheppard) [reply Content.Client/Audio/BackgroundAudioSystem.cs]: Yes. fixed

**Pull request merged**